config/initializers/diaspora_federation.rb

```diff
@@ -0,0 +1,8 @@
+# configure the federation engine
+DiasporaFederation.configure do |config|
+  # the pod url
+  config.server_uri = AppConfig.pod_uri
+
+  # the class to be used for a person
+  config.person_class = Person.to_s
```

Comment:
The Rails documentation says it should be done with a String: http://guides.rubyonrails.org/engines.html#configuring-an-engine (the red block)

> It's very important here to use the String version of the class, rather than the class itself. If you were to use the class, Rails would attempt to load that class and then reference the related table. This could lead to problems if the table wasn't already existing. Therefore, a String should be used and then converted to a class using constantize in the engine later on.

So, because it's red, I thought it's important to use a String here ;) But I can move the .to_s to the gem and still hold it as a String in the gem.